SR2022/2021-11-10-kickstart-details.md

```diff
@@ -0,0 +1,29 @@
+---
+to: Student Robotics 2022 registered teams
+subject: Kickstart Details
+---
+
+Hi teams
+
+Only a couple of days left until Saturday’s Kickstart event, so here are some more details on what to expect and what we need from you.
+
+## Kits
```

Comment:
Suggestion: If the kit section is moved underneath "Kickstart", then the "Kickstart" heading can be removed entirely, and the email may flow a little more nicely?

That said, the downside is that the mention of kit is lower in the email, when in fact it's very important